man/dcp.1

```diff
@@ -60,6 +60,17 @@ The default chunksize is 64MB.
 .UNINDENT
 .INDENT 0.0
 .TP
+.B \-\-xattrs WHICH
+Copy extended attributes ("xattrs") from source files to target files.  WHICH
+determines which xattrs are copied.  Options are to copy no xattrs, all xattrs,
+xattrs not excluded by /etc/xattr.conf, or all xattrs except those which have
+special meaning to lustre.  Certain xattrs control lustre features on a
+file-by-file basis, such as how the file data is distributed across Lustre
+servers.  Values must be in {none, all, libattr, non-lustre}.  The default is
+non-lustre.
+.UNINDENT
+.INDENT 0.0
+.TP
```

Comment:
Would you please move this to ``doc/rst/dcp1.rst``.  We auto-generate these man pages from the rst source files.

Let's also capitalize "Lustre" in the user-facing text.

Reply:
I can help generate the man pages from there.

Reply:
Cool, I didn't notice the rst files.  So much easier to read and edit than nroff or whatever that is.